plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -709,6 +708,20 @@ public void testAddColumnConcurrently()
         throw new SkipException("TODO");
     }
 
+    @Test
+    public void testSetColumnTypeNonLowercaseColumn()
+    {
+        String tableName = "test_non_lowercase_column" + randomNameSuffix();
```

Comment:
Another test that is currently missing are overflow tests.
What happens if the column has values which are greater than what  the destination column type can hold? Are they being truncated/ does the operation fail?